src/grizzly/services/webtransport/core.py

```diff
@@ -71,8 +71,8 @@ async def is_ready(self) -> None:
         """Wait until the service is ready"""
         await _connect_to_server("127.0.0.1", self.port)
 
-    def start(self) -> None:
-        """Start the server."""
+    def start(self, timeout: int = 5) -> None:
```

Comment:
Is 5 seconds enough? Would 30 be better to avoid false positives? Or just wait and if we see issues then bump it up?